File: iosApp/iosApp/Presentation/Main/CafeManagement/MenuGoodsEdit/MenuGoodsEditUiState.swift
//
//  MenuGoodsEditUiState.swift
//  ConCafe
//
//  Created by 홍희표 on 2026/03/11.
//

import Foundation
import Shared

struct MenuGoodsEditUiState {
    var detail: CafeDetail?
    var isLoading = true
    var isSaving = false
    var isEditMode = false
    var itemName = ""
    var price = ""
    var selectedCategoryId = "drink"
    var description = ""
    var isInStock = true
    var imageUrl: String?
    var infoMessageKey: String?
}

File: composeApp/src/commonMain/kotlin/com/hhp227/concafe/presentation/main/cafemanagement/menugoodsedit/MenuGoodsEditUiState.kt
package com.hhp227.concafe.presentation.main.cafemanagement.menugoodsedit

data class MenuGoodsEditUiState(
    val isLoading: Boolean = true,
    val isEditMode: Boolean = false,
    val itemName: String = "",
    val price: String = "",
    val selectedCategoryId: String = "drink",
    val description: String = "",
    val isInStock: Boolean = true,
    val imageUrl: String? = null,
    val infoMessageKey: String? = null
)
